### ISSUE TITLE
refactor(docs): rewrite docs to be tsdoc based

### ISSUE THREAD

**Issue by dansaadati** [googleapis/nodejs-storage#2111]
We're seeing missing links and other issues on docs hosted at [https://cloud.google.com/nodejs/docs/reference/storage/latest](https://cloud.google.com/nodejs/docs/reference/storage/latest). We should rewrite docs to be tsdoc based, which is compatible with Cloud RAD. From there, we can evaluate what other issues should be addressed.

**Comment by ddelgrosso1** [comment]
@dansaadati is this still planned to be rewritten?

**Comment by dansaadati** [comment]
> We're seeing missing links and other issues on docs hosted at https://cloud.google.com/nodejs/docs/reference/storage/latest. We should rewrite docs to be tsdoc based, which is compatible with Cloud RAD. From there, we can evaluate what other issues should be addressed.

Yes, we've been doing work with our Cloud RAD Node.js generator to address bugs/improve quality. Once that looks better, we'll rewrite docs to be TSDoc-based.
